index.js

```diff
@@ -141,130 +141,4 @@ exports.get = function (archiveFile, archive) {
-        }
-
-        event = sort(event);
-        console.log(args.json ? JSON.stringify(event, undefined, '  ') : event);
-      }
-
-      logStats() {
-        // order events by bits parsed and output them from most to least
-        Object.keys(this.eventStats).sort((b, a) => {
-          return this.eventStats[b][1] > this.eventStats[a][1];
-        }).forEach(name => {
-          const stat = this.eventStats[name];
-          console.log(`${name}, ${stat[0]}, ${stat[1]}`);
-        });
-      }
-    }
-
-    const yargs = require('yargs')
-                    .usage('usage: heroprotocol.js replayFile [--help] [--header] [--details] [--initdata] [--gameevents] [--messageevents] [--trackerevents] [--attributeevents] [--stats]')
-                    .demand(1)
-                    .option('h', { alias: 'help', type: 'boolean', desc: 'show this help' })
-                    .option('H', { alias: 'header', type: 'boolean', desc: 'print protocol header' })
-                    .option('d', { alias: 'details', type: 'boolean', desc: 'print protocol details' })
-                    .option('i', { alias: 'initdata', type: 'boolean', desc: 'print protocol initdata' })
-                    .option('g', { alias: 'gameevents', type: 'boolean', desc: 'print game events' })
-                    .option('m', { alias: 'messageevents', type: 'boolean', desc: 'print message events' })
-                    .option('t', { alias: 'trackerevents', type: 'boolean', desc: 'print tracker events' })
-                    .option('a', { alias: 'attributeevents', type: 'boolean', desc: 'print attribute events' })
-                    .option('s', { alias: 'stats', type: 'boolean', desc: 'print stats' })
-                    .option('json', { type: 'boolean', desc: 'prints in JSON format' });
-    const args = yargs.argv;
-
-    if (args.help) {
-      yargs.showHelp();
-      process.exit();
-    }
-
-    const archive = exports.open(args._[0]);
-
-    if (archive instanceof Error) {
-      console.log(archive.error);
-      process.exit(1);
-    }
-
-    const logger = new EventLogger();
-
-    if (args.header) {
-      logger.log(archive.data[HEADER]);
-    }
-
-    if (!archive.protocol) {
-      console.log('Unsupported base build:', archive.baseBuild);
-      process.exit(1);
-    }
-
-    if (args.details) {
-      logger.log(archive.get(DETAILS));
-    }
-
-    if (args.initdata) {
-      const data = archive.get(INITDATA);
-      logger.log(data.m_syncLobbyState.m_gameDescription.m_cacheHandles);
-      logger.log(data);
-    }
-
-    if (args.gameevents) {
-      logger.log(archive.get(GAME_EVENTS));
-    }
-
-    if (args.messageevents) {
-      logger.log(archive.get(MESSAGE_EVENTS));
-    }
-
-    if (args.trackerevents) {
-      logger.log(archive.get(TRACKER_EVENTS));
-    }
-
-    if (args.attributeevents) {
-      logger.log(archive.get(ATTRIBUTES_EVENTS));
-    }
-
-    if (args.stats) {
-      logger.logStats();
-    }
-
-  })();
-}
+};
```

Comment:
Keep the end of file empty line. Maybe update your editor preference so it doesn't remove it automatically on save.